core/package.json

```diff
@@ -57,7 +57,7 @@
     "jest-cli": "^26.4.1",
     "np": "^6.4.0",
     "pixelmatch": "4.0.2",
-    "puppeteer": "^7.0.1",
+    "puppeteer": "^10.4.0",
```

Comment:
The install works for me 🎉 Should we update these via `npm i -D puppeteer@10` to keep `package-lock.json` up to date?